src/themes/ec/variables/_list-illustration.scss

```diff
@@ -4,9 +4,13 @@
 $list-illustration: (
```

Comment:
There are still fonts using static size definition in the component. If we have time, it may be worth using the responsive fonts for them